content/en/docs/ops/common-problems/security-issues/index.md

```diff
@@ -131,19 +135,19 @@ you ensure Istiod is working as expected:
 
 1. Start your browser and open the `ControlZ` page at `http://127.0.0.1:9876/scopez/`.
 
-1. Change the `rbac` Output Level to `debug`.
+1. Change the `authorization` Output Level to `debug`.
```

Comment:
can you also update the commands of step 1 and 2 to use `istioctl dashboard controlz $(kubectl -n istio-system get pods -l app=istiod -o jsonpath='{.items[0].metadata.name}').istio-system`

Reply:
fixed